src/components/ChatEntry.js

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
-import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import './ChatEntry.css';
 
 const ChatEntry = (props) => {
+  const likedStatus = props.liked ? '❤️' : '🤍';
+  const senderLocation = props.sender1 === props.sender ? 'local' : 'remote';
```

Comment:
Nice job using the liked and location value coming from props to update the component display. There's no need to track these with the local state, since any time we update the state (using the supplied callback) the component data will update, which we will receive through updated props.